modules/node_modules/@ncigdc/dux/cart.js

```diff
@@ -207,7 +203,9 @@ function reducer(state = initialState, action) {
           access: file.access,
           file_id: file.file_id,
           file_size: file.file_size,
-          projects: file.projects || file.cases.map(c => c.project.project_id),
+          projects: file.projects
+            || file.cases.map(c => c.project.project_id)
+            || file.cases.hits.edges.map(c => c.node.project.project_id),
```

Comment:
if `file.cases` is not an array then this will cause an exception. you'll need to check the values first.